src/showAPopUpScreen.ts

```diff
@@ -0,0 +1,15 @@
+export const showAPopUpScreen = (screenToDisplay: HTMLDivElement, startingPosition: string) => {
+    // display the help screen
+     screenToDisplay.style.display = startingPosition;
+    // function which gets called when the exit button is clicked
+    if (screenToDisplay.querySelector('.exitBtn')) {
```

Comment:
three times this: `screenToDisplay.querySelector('.exitBtn')`
please, use variable to minimise searching through DOM :)